apps/meteor/app/apps/server/bridges/rooms.ts

```diff
@@ -160,11 +166,24 @@ export class AppRoomBridge extends RoomBridge {
 	protected async update(room: IRoom, members: Array<string> = [], appId: string): Promise<void> {
 		this.orch.debugLog(`The App ${appId} is updating a room.`);
 
-		if (!room.id || !(await Rooms.findOneById(room.id))) {
+		const originalRoom = await Rooms.findOneById(room.id);
+		if (!room.id || !originalRoom) {
 			throw new Error('A room must exist to update.');
 		}
 
-		const rm = await this.orch.getConverters()?.get('rooms').convertAppRoom(room);
+		let rm = await this.orch.getConverters()?.get('rooms').convertAppRoom(room);
+
+		if (isOmnichannelRoom(originalRoom)) {
+			// We shouldn't allow to mess up with a room's closed state via apps. Only via livechat bridge
```

Comment:
shouldn't we outright throw an error if one try to do this? silent failure can be *really* annoying

Reply:
Thought so, but felt throwing a _new_ error on update could be a breaking change. Silent here since people shouldn't be touching any of the props listed, so we'll assume they did by mistake and correct the room state.

Changing closing status, type of livechat rooms & open/close flag is not supported (and never was)

Reply:
Yea since we never aimed to support it I thought it would not be that big of a deal adding it (even though technically it is a breaking change). But if you think it is not that big of a deal I am fine with it.
(maybe some logging would be useful so we can have visibility, when a customer is trying
 to close a room through update and it not working)

Reply:
I agree, we can also add in app's logs, that the app tried to update one of the **restricted fields** and bla bla

Reply:
Right, good call